packages/libs/src/sdk/services/Solana/programs/BaseSolanaProgramClient.ts

```diff
@@ -90,14 +105,41 @@ export class BaseSolanaProgramClient {
       instructions,
       feePayer,
       recentBlockhash,
-      addressLookupTables = []
+      addressLookupTables = [],
+      priorityFee
     } = await parseParams('buildTransaction', BuildTransactionSchema)(params)
 
     if (!recentBlockhash) {
       const res = await this.connection.getLatestBlockhash()
       recentBlockhash = res.blockhash
     }
 
+    if (priorityFee !== undefined) {
```

Comment:
nit: why `!== undefined` instead of `!!priorityFee`?

Reply:
zero is valid

Reply:
oh wait, forgot I made this an object